src/backend/main/go.backends/index/elasticsearch/messages.go

```diff
@@ -174,6 +175,24 @@ func (es *ElasticSearchBackend) GetMessagesRange(filter objects.IndexSearch) (me
 	return messages, totalFound, nil
 }
 
+// return false if message not found or error
+func (es *ElasticSearchBackend) SeekMessageByExternalRef(userID, externalMessageID string) (bool, error) {
```

Comment:
Remove this function ?

Reply:
useless for now, but code is written, could we keep it for later ?

Reply:
I'm not sure at all it will be used later and we can find it in git history if really needed